storage/client/client.go

```diff
@@ -256,29 +189,19 @@ func (c *StorageClient) Transactions(ctx context.Context, p apiTypes.GetConsensu
 	for rows.Next() {
 		var t Transaction
 		var code uint64
-		var feeNum pgtype.Numeric
 		if err := rows.Scan(
 			&t.Block,
 			&t.Index,
 			&t.Hash,
 			&t.Sender,
 			&t.Nonce,
-			&feeNum,
+			&t.Fee,
```

Comment:
Just checking - this works for the custom db types (`UINT63` + `UINT31`) now?

Reply:
I believe it always worked for those. The new thing is that it now works for `UINT_NUMERIC` (and `NUMERIC` variants in general), thanks to commit [`e7de6a4`](https://github.com/oasisprotocol/oasis-indexer/pull/297/commits/e7de6a4172b2eedaca4cfd643c708ca23bf6c7c2)